src/wp-includes/version.php

```diff
@@ -37,7 +37,7 @@
  *
  * @global string $required_php_version
  */
-$required_php_version = '5.6.20';
+$required_php_version = '7.2';
```

Comment:
```suggestion
$required_php_version = '7.2.0';
```
(or whatever patch version will be decided upon in the end, but it MUST be a three-tier version number)